### DISCUSSION TITLE
GriefPrevention Claim Blocks Limits

### DISCUSSION THREAD

**Discussion by AbdulGhafoor01** [GriefPrevention/GriefPrevention#1427] [Q&A] [answered]
Is there any way I can allow a specific rank to have more claim blocks limit?
Old Config:
-----------------------------------------
    Max Accrued Claim Blocks:
      Default: 3333
-----------------------------------------
I want Config:
-----------------------------------------
    Max Accrued Claim Blocks:
      Default: 3333
      VIP: 4500
      mvp: 5000
      legend: 6500
-----------------------------------------
I use ↑ Config but it doesn't work. When I run the server it auto change the config with the old one

**Comment by wmattiso** [comment]
Are those all roles set up in your permissions plugin?

**Reply by AbdulGhafoor01** [reply]
yes

**Comment by Jikoo** [comment, marked as answer]
GP does not offer this as a core feature. The addon [GPAccruals](https://github.com/GriefPrevention/GriefPrevention/discussions/2075) allows for permission-based accrual rates and maximums.

**Reply by AbdulGhafoor01** [reply]
ok

**Reply by ivaniraldi** [reply]
bro the link isn't working im getting crazy

**Reply by Evidentsinger14** [reply]
You can find the addons [here](https://github.com/GriefPrevention/GriefPrevention/discussions/categories/addons)

**Reply by Jikoo** [reply]
> bro the link isn't working im getting crazy

I've updated the link, didn't realize Robo deleted the wiki page when creating the addons category.

**Reply by ivaniraldi** [reply]
thanks man ur amazing
